src/containers/Nav.js

```diff
@@ -0,0 +1,41 @@
+import React, {Component} from 'react' 
+import { AuthStore } from '../repos/UserRepo';
+import {NavLink, withRouter} from 'react-router-dom'
+import { connect } from 'react-redux';
+
+//TODO make responsivity better
+class Nav extends Component {
+    render() {
+        if (AuthStore.isAuthenticated) {
+            return (
+            <nav className="navbar navbar-expand-lg navbar-light bg-light" >
+                <NavLink className="navbar-brand d-none d-lg-inline" to="/">NuTracker</NavLink>
```

Comment:
NavLink is component provided by react router it automatically adds the css class active to the current location